examples/features/objects.effekt.md

```diff
@@ -6,48 +6,80 @@ permalink: docs/tutorial/objects
 
 # Objects
 
-Given an effect interface;
+Effekt features a simple object-system. Let us assume the following interface:
 
 ```
-interface Exception {
-  def throw(msg: String): Nothing
+interface Counter {
+  def increment(): Unit
+  def get(): Int
 }
 ```
 
-besides implementing a handler for it, there is also another way of defining the semantics of an effect:
+Interfaces are [_computation types_](./computation.md) and we can write a program that expects a counter to work on:
```

Comment:
Will such a link work on the website? 🤔

Reply:
In the parser case study, we link to the lexer case study like this:
```
# Parsing
In this case study, we show how to implement a parser, using the lexer from the
[Lexer case study](lexer).
```

So here, it should probably be:
```suggestion
Interfaces are [_computation types_](computation) and we can write a program that expects a counter to work on:
```
?

Reply:
Ditto for the links in captures 👀